tensorflow/python/ops/clip_ops_test.py

```diff
@@ -96,6 +96,56 @@ def testClipIndexedSlicesByNorm(self):
     # Axes == [0, 1]
     self._testClipIndexedSlicesByNorm(values, indices, shape, 4.0, [0, 1, 2])
 
+  def _testClipTensorByValue(self, inputs, clip_value_min, clip_value_max, \
```

Comment:
Could you move tests to [tensorflow/python/kernel_tests/clip_ops_test.py](https://github.com/tensorflow/tensorflow/blob/1a22ddc4e545ec76c0b7876788059f5a69406bca/tensorflow/python/kernel_tests/clip_ops_test.py#L54) ?

Reply:
And also, please remove those backslashes at the end of line :-)

Reply:
Done~